micro_brewery/data/website_view.xml

```diff
@@ -149,13 +149,6 @@
             </t>
         </field>
     </record>
-
-    <template id="micro_brewery_custom_appointment_heading" inherit_id="appointment.appointment_info">
-        <xpath expr="//h4[hasclass('pb-3')]" position="replace">
-            <h4 class="pb-3">Schedule <span style="color:#714b67;">Visit of Brewery</span></h4>
-        </xpath>
-    </template>
```

Comment:
Isn't this custo interesting? :thinking:

Reply:
Yes but it implies studio.